test/jet/parse_opts_test.clj

```diff
@@ -0,0 +1,74 @@
+(ns jet.parse-opts-test
+  (:require
+   [clojure.test :as test :refer [are deftest is testing]]
+   [jet.main :refer [parse-opts]]))
+
+(defn compare-opts
+  "Compare `short` & `long` options."
+  [short long]
+  (= (parse-opts short)
+     (parse-opts long)))
+
+(deftest parse-opts-test
```

Comment:
All the options are covered with the tests below, they aren't in alphabetical order. Should they be?

Reply:
That's fine.